youtube_dl/extractor/myspace.py

```diff
@@ -204,3 +204,46 @@ def _real_extract(self, url):
             'title': self._og_search_title(webpage),
             'entries': entries,
         }
+
+
+class MySpaceArtistSongsIE(InfoExtractor):
+    IE_NAME = 'MySpace:artist:songs'
+    _VALID_URL = r'https?://myspace\.com/(?P<artist>[^/]*)/music/songs'
+
+    _TESTS = [{
+        'url': 'https://myspace.com/starset2/music/album/transmissions-19455773',
+        'url': 'https://myspace.com/ryanlewismusic/music/songs',
+        'info_dict': {
+            'title': 'RYAN LEWIS',
+        },
+        'playlist_count': 8,
+    }, {
+        'url': 'https://myspace.com/studio99/music/songs/',
+        'info_dict': {
+            'title': 'Studio 99 IS CLOSED!! R.I.P.',
+        },
+        'playlist_count': 4,
+    }]
+
+    def _real_extract(self, url):
+        mobj = re.match(self._VALID_URL, url)
+        display_id = mobj.group('artist')
+        webpage = self._download_webpage(url, display_id)
+        tracks_paths = re.findall(r'<meta itemprop="url" content ="(.*?)">', webpage)
+        if not tracks_paths:
+            raise ExtractorError(
+                '%s: No songs found, try using proxy' % display_id,
```

Comment:
How are missing songs related to using proxy? https://myspace.com/mekowilliamssho/music/songs have no songs but suggests using proxy. This check should be removed completely. youtube-dl will handle this on it's own.